packages/snage/src/note/parser.ts

```diff
@@ -43,7 +45,7 @@ export const readNote = (
     pipe(
         readFile(fileName),
         TE.mapLeft((e) => [e]),
-        TE.map((content) => parseRawNote(content, fileName)),
+        TE.chainEitherK((content) => E.either.mapLeft(parseRawNote(content, fileName), (err) => [err])),
```

Comment:
könnte man E.Bifunctor.mapLeft benutzen? E.either ist deprecated. zumindest die tests in parser.test.ts sind dann immernoch grün \:o

Reply:
Jo, ich glaub ich muss dann nochmal in einem separaten PR alle deprecations fixen.